specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-11-01-preview/DynamicSummaries.json

```diff
@@ -0,0 +1,398 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Insights",
+    "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
```

Comment:
> "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider" [](http://example.com/codeflow?start=18&length=86)

This is a doc of the API. So the description should describe not the 'api spec', but the API

Reply:
This is a common description for all the services under securityInsights/resourceManager and DynamicSummaries followed the same standard. they all have same description